src/components/movieForm/MovieForm.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./MovieForm.scss";
+import {Dialog} from "../index";
+
+const MovieForm = ({formTitle, submitCallback, initialMovie, showModal}) => {
+    let formLabels = {
+        title: 'title',
+        release_date: 'Release date',
+        url: 'movie url',
+        rating: 'rating',
+        genre: 'genre',
+        runtime: 'runtime',
+        overview: 'overview'
+    };
+
+    const genreOptions = ["all", "comedy", "drama", "detective"];
+
+    Object.keys(formLabels).forEach((key) => {
+        formLabels[key] = formLabels[key].toUpperCase();
+    });

```

Comment:
From line 6 to this line, variables will be created every time when the props are updated. Any constants outside the component?